test/nuxeo-date-picker.html

```diff
@@ -112,6 +112,13 @@
       expect(getInput(element).value).to.be.equal('');
     });
 
+    test('the input changes takes default time into account', () => {
+      element.defaultTime = '14:35:19';
+      getInput(element).value = '2003-02-20';
+      expect(moment(element.value).minute()).to.be.equal(35);
```

Comment:
Should we also define expectation for hour?

Reply:
It is rather cautious because we don't know the timezone of the machine where it is going to run :D 

Asserting minutes and seconds should be sufficient to makes sure `defaultTime` is applied.